### ISSUE TITLE
Simple Style Creator and Editor

### ISSUE THREAD

**Issue by wepache** [SubhadeepJasu/Ensembles#77]
It would be nice if there was a way to create your own styles within the program. The easiest way for the user would be an editor like major Digital Audio Workstations use already, with a Piano roll, and midi editor, etc.

**Comment by SubhadeepJasu** [comment]
Yeah this all ties into the ability to write MIDI files. I have this planned already. This would improve the recorder as well.
